web/src/org/openmrs/module/chica/web/ManualCheckinSSNMRN.java

```diff
@@ -38,8 +31,6 @@
 	private static final String XML_FALSE = "false";
 	private static final String XML_SESSION_RESULT_START = "<sessionResult>";

```

Comment:
Thoughts on the naming of this? session doesn't really seem to fit what we are checking here.